index.html

```diff
@@ -80,6 +80,7 @@ <h1>YouTube</h1>
 
 
     <main>
+        <section>
         <div id="video-container">
             <div class="video-template">
                 <img class="video-image" src="https://picsum.photos/400/200" alt="">
```

Comment:
## Images are missing the alt reference

Please add the alt tag for better SEO and good html practices.